oauth2_provider/locale/es/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,197 @@
+
+#: oauth2_provider/templates/oauth2_provider/application_confirm_delete.html:12
+#: oauth2_provider/templates/oauth2_provider/authorize.html:29
+msgid "Cancel"
+msgstr "Cancelar"
+
+#: oauth2_provider/templates/oauth2_provider/application_confirm_delete.html:13
+#: oauth2_provider/templates/oauth2_provider/application_detail.html:38
+#: oauth2_provider/templates/oauth2_provider/authorized-token-delete.html:7
+msgid "Delete"
+msgstr "Eliminar"
+
+#: oauth2_provider/templates/oauth2_provider/application_detail.html:10
+msgid "Client id"
+msgstr "Identificador de cliente"
+
+#: oauth2_provider/templates/oauth2_provider/application_detail.html:15
+msgid "Client secret"
+msgstr "Secreto de cliente"
+
+#: oauth2_provider/templates/oauth2_provider/application_detail.html:20
+msgid "Client type"
+msgstr "Tipo de cliente"
+
+#: oauth2_provider/templates/oauth2_provider/application_detail.html:25
+msgid "Authorization Grant Type"
+msgstr "Tipo de acceso de autorización"
+
+#: oauth2_provider/templates/oauth2_provider/application_detail.html:30
+msgid "Redirect Uris"
+msgstr "Uris de redirección"
+
+#: oauth2_provider/templates/oauth2_provider/application_detail.html:36
+#: oauth2_provider/templates/oauth2_provider/application_form.html:35
+msgid "Go Back"
+msgstr "Volver"
+
+#: oauth2_provider/templates/oauth2_provider/application_detail.html:37
+msgid "Edit"
+msgstr "Editar"
+
+#: oauth2_provider/templates/oauth2_provider/application_form.html:9
+msgid "Edit application"
+msgstr "Editar aplicación"
+
+#: oauth2_provider/templates/oauth2_provider/application_form.html:37
+msgid "Save"
+msgstr "Guardar"
+
+#: oauth2_provider/templates/oauth2_provider/application_list.html:6
+msgid "Your applications"
+msgstr "Tus aplicaciones"
+
+#: oauth2_provider/templates/oauth2_provider/application_list.html:14
+msgid "New Application"
+msgstr "Nueva aplicación"
+
+#: oauth2_provider/templates/oauth2_provider/application_list.html:17
+msgid "No applications defined"
+msgstr "No hay aplicaciones definidas"
+
+#: oauth2_provider/templates/oauth2_provider/application_list.html:17
+msgid "Click here"
+msgstr "Click aquí"
+
+#: oauth2_provider/templates/oauth2_provider/application_list.html:17
+msgid "if you want to register a new one"
+msgstr "si quiere regitrar una nueva"
+
+#: oauth2_provider/templates/oauth2_provider/application_registration_form.html:5
+msgid "Register a new application"
+msgstr "Registrar una nueva aplicación"
+
+#: oauth2_provider/templates/oauth2_provider/authorize.html:8
+#: oauth2_provider/templates/oauth2_provider/authorize.html:30
+msgid "Authorize"
+msgstr "Autorizar"
+
+#: oauth2_provider/templates/oauth2_provider/authorize.html:17
+msgid "Application requires the following permissions"
+msgstr "La aplicación requiere los siguientes permisos"
+
+#: oauth2_provider/templates/oauth2_provider/authorized-token-delete.html:6
+msgid "Are you sure you want to delete this token?"
+msgstr "¿Está seguro de que quiere eliminar este token?"
```

Comment:
is there a difference between estas and esta? Translation of you is a bit all over the place, but I'm just nitpicking.